browser/ui/views/tabs/brave_tab_container.h

```diff
@@ -29,6 +29,12 @@ class BraveTabContainer : public TabContainerImpl {
   void EnterTabClosingMode(absl::optional<int> override_width,
                            CloseTabSource source) override;
   bool ShouldTabBeVisible(const Tab* tab) const override;
+  void StartInsertTabAnimation(int model_index) override;
+  void RemoveTab(int index, bool was_active) override;
+  void OnTabCloseAnimationCompleted(Tab* tab) override;
+
+ private:
+  base::flat_set<Tab*> closing_tabs_;
```

Comment:
Can we guarantee this set is empty in dtor?

Reply:
Good idea. I added a DCHECK in dtor.